CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.5.1 FATAL_ERROR)
 #============================================================================
 # Initialize the project
 #============================================================================
-project(ignition-fuel_tools4 VERSION 4.9.0)
+project(ignition-fuel_tools4 VERSION 4.10.0)
```

Comment:
Since there are no API changes, I think this should be 4.9.1

Reply:
Will change it